### DISCUSSION TITLE
听书桌面常驻文本

### DISCUSSION THREAD

**Discussion by Yanick112** [read-cat/read-cat#16] [General]
听书桌面常驻文本

**Comment by Yanick112** [comment]
这个软件实在是泰裤辣，我想帮忙一起完善，虽然实力很糟糕，如题功能大大有计划做吗，从何下手，可以是插件的形式吗，我能帮上忙吗

**Comment by KitWingPunGu** [comment]
你说的这功能，我也很需要。我平常喜欢听书，但现在有的朗读软件朗读的发音太机械了，这软件实在太好了。就是类似歌词朗读的时候同步文字。希望有这个功能，感谢。
